canonical-websites/www.ubuntu.com
# Add basic modal contact us on core page

## Done
Add basic modal contact us on core page

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Go to /core
- Open the console and enter `dynamicContactUs();`
- Click any of the contact buttons and see the modal is displayed
- Click through the modal and check it goes back and forward
- Once complete check the Network tab has a form submission
- Check the contents of the request is correct
